### PR TITLE
Document danger of using COPY and ADD instructions

### PR DESCRIPTION
**Pull request by fkorotkov**

### CODE REVIEW

File: docs/guide/docker-builder-vm.md [hunk 1]
```diff
@@ -90,7 +90,12 @@ Cirrus CI will check if a container was already built, and if so, Cirrus CI will
 Under the hood, for every `Dockerfile` that is needed to be built, Cirrus CI will create a Docker Builder task as a dependency. 
 You will see such `build_docker_image_HASH` tasks in the UI.
 
-!!! info "Using with private GKE clusters"
+!!! warning "Danger of using `COPY` and `ADD` instructions"
+    Cirrus doesn't include files added or copied into a container image in the cache key. This means that for a public repository
+    a potential bad actor can create a PR with malicious scripts included into a container, wait for it to be cached and then
+    reset the PR so it looks harmful.
```
Comment (RDIL) [inline docs/guide/docker-builder-vm.md:96]: That doesn't sound quite right... you mean reset the PR so it doesn't look harmful?

Comment (fkorotkov) [reply docs/guide/docker-builder-vm.md:96]: Reset PR branch. Like modify scripts that are added to not look harmful.

Comment (real-or-random) [reply docs/guide/docker-builder-vm.md:96]: I think @RDIL meant "s/harmful/not harmful/". Maybe it's good to add one or two sentences about the consequences, e.g., what a malicious script could do.

Comment (fkorotkov) [reply docs/guide/docker-builder-vm.md:96]: Oh, yeah, some other folks noticed it and I fixed the harmfull thing in 51474e16c3939a0608b5acb681d6ab21b9ce70d8

**Pull request merged**